### PR TITLE
Update nimble_publisher.md to phoenix version 1.7

### PR DESCRIPTION
**Pull request by scubass**

### CODE REVIEW

File: lessons/en/misc/nimble_publisher.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 %{
-  version: "1.0.2",
+  version: "2.0.0",
```
Comment (doomspork) [inline lessons/en/misc/nimble_publisher.md:2]: @scubass can provide some additional context on why we need this version change?

Comment (scubass) [reply lessons/en/misc/nimble_publisher.md:2]: Phoenix changed a lot and is not compatible with the other version code, so I thought that the version should reflect the major update to the library.

Comment (doomspork) [reply lessons/en/misc/nimble_publisher.md:2]: This version reflects the _lesson's_ version to notify translators when changes need to be made.

**Comment by doomspork** [comment]
This change doesn't appear to be necessary so I'm closing this for now. The lesson versions reflect the version of the lesson content. We shouldn't need to update that without updating the content itself.

**Pull request closed without merging**